cli/cppcheckexecutor.cpp

```diff
@@ -278,6 +278,7 @@ static void print_stacktrace(FILE* output, bool demangling, int maxdepth, bool l
         char **symbolStringList = backtrace_symbols(callstackArray, currentdepth);
         if (symbolStringList) {
             fputs("Callstack:\n", output);
+			char demangle_buffer[2048]= {0};
```

Comment:
indentation.. if you can configure your editor to always replace tabs with spaces feel free to do that..

Reply:
I guess the scope of `demangle_buffer` can be reduced. since I don't know the exact semantics of the `__cxa_demangle` it might feel a bit more robust to put it in the loop so information can not leak into the next iteration.

Reply:
This change already landed on the `main` branch with proper formatting.

I checked the documentation of the function and it will only return a valid pointer to a NUL-terminated string if it was successful.

Reply:
Actually - my IDE is currently configured to not use Tabs but it is somehow not applied...